Dockerfile

```
@@ -1,151 +1,121 @@
 ARG ARCH=amd64
-ARG NPM_VERSION=6.9.0
-
-# The node version here should match the version of the runtime image which is
-# specified in the base-image subdirectory in the project
-FROM balenalib/raspberry-pi-node:10-run as rpi-node-base
-FROM balenalib/armv7hf-node:10-run as armv7hf-node-base
-FROM balenalib/aarch64-node:10-run as aarch64-node-base
-RUN [ "cross-build-start" ]
-RUN sed -i '/security.debian.org jessie/d' /etc/apt/sources.list
-RUN [ "cross-build-end" ]
-
-FROM balenalib/amd64-node:10-run as amd64-node-base
-RUN echo '#!/bin/sh\nexit 0' > /usr/bin/cross-build-start && chmod +x /usr/bin/cross-build-start \
-	&& echo '#!/bin/sh\nexit 0' > /usr/bin/cross-build-end && chmod +x /usr/bin/cross-build-end
-
-FROM balenalib/i386-node:10-run as i386-node-base
-RUN echo '#!/bin/sh\nexit 0' > /usr/bin/cross-build-start && chmod +x /usr/bin/cross-build-start \
-	&& echo '#!/bin/sh\nexit 0' > /usr/bin/cross-build-end && chmod +x /usr/bin/cross-build-end
-
-FROM balenalib/i386-nlp-node:6-jessie as i386-nlp-node-base
-RUN echo '#!/bin/sh\nexit 0' > /usr/bin/cross-build-start && chmod +x /usr/bin/cross-build-start \
-	&& echo '#!/bin/sh\nexit 0' > /usr/bin/cross-build-end && chmod +x /usr/bin/cross-build-end
-
-# Setup webpack building base images
-# We always do the webpack build on amd64, cause it's way faster
-FROM amd64-node-base as rpi-node-build
-FROM amd64-node-base as amd64-node-build
-FROM amd64-node-base as armv7hf-node-build
-FROM amd64-node-base as aarch64-node-build
-FROM amd64-node-base as i386-node-build
-FROM balenalib/amd64-node:6-build as i386-nlp-node-build
-##############################################################################
-
-FROM $ARCH-node-build as node-build
+ARG NODE_VERSION=10.18.0
```

v10.19.0 would make more sense:
```suggestion
ARG NODE_VERSION=10.19.0
```